src/queryWizard/AccordionWizard.js

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import React from 'react'
+import DefaultIcon from '../DefaultIcon'
+import WizardStep from './WizardStep'
+
+let splitKeys = _.curry((keys, obj) => [_.pick(keys, obj), _.omit(keys, obj)])
```

Comment:
I think you might be looking for [`_.partition`](https://lodash.com/docs/4.17.11#partition)

Reply:
I was - not sure how I missed it 🤦‍♀

Reply:
actually, `_.partition` won't work - it only passes `value` to the predicate, so we still need a new function for splitting on keys.

(I also played around with `reduce` to avoid mapping twice, but it's significantly less readable for what's likely a completely unnoticeable performance gain, so I guess I'll just stick with `_.pick` and `_.omit`)